packages/core/components/DataTable/helpers/boxplotCellMatrix.tsx

```diff
@@ -28,30 +28,31 @@ const boxplotCellMatrix = ({ rows, config }): CellMatrix => {
     return resolvedName
   }
   let resolveCell = (rowid, plot) => {
-    if (Number(rowid) === 0) return true
-    if (Number(rowid) === 1) return plot.columnMax
-    if (Number(rowid) === 2) return plot.columnThirdQuartile
-    if (Number(rowid) === 3) return plot.columnMedian
-    if (Number(rowid) === 4) return plot.columnFirstQuartile
-    if (Number(rowid) === 5) return plot.columnMin
-    if (Number(rowid) === 6) return plot.columnCount
-    if (Number(rowid) === 7) return plot.columnSd
-    if (Number(rowid) === 8) return plot.columnMean
-    if (Number(rowid) === 9) return plot.columnOutliers.length > 0 ? plot.columnOutliers.toString() : '-'
-    if (Number(rowid) === 10) return plot.values.length > 0 ? plot.values.toString() : '-'
+    if (Number(rowid) === 0) return plot.columnMax
+    if (Number(rowid) === 1) return plot.columnThirdQuartile
+    if (Number(rowid) === 2) return plot.columnMedian
+    if (Number(rowid) === 3) return plot.columnFirstQuartile
+    if (Number(rowid) === 4) return plot.columnMin
+    if (Number(rowid) === 5) return plot.columnCount
+    if (Number(rowid) === 6) return plot.columnSd
+    if (Number(rowid) === 7) return plot.columnMean
+    if (Number(rowid) === 8) return plot.columnOutliers.length > 0 ? plot.columnOutliers.toString() : '-'
+    if (Number(rowid) === 9) return plot.values.length > 0 ? plot.values.toString() : '-'

```

Comment:
can you create helper functions for whatever this is doing? and reuse it on line 39 as well?